src/test/java/CustomUserDAOTests.java

```diff
@@ -0,0 +1,48 @@
+import TTT.databaseUtils.CustomUserDAO;
+import TTT.users.CustomUser;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class CustomUserDAOTests {
+
+    CustomUserDAO testObject = new CustomUserDAO();
+    CustomUser testCustomUser = new CustomUser("test@mail.com","123");
+
+    @Test
+    public void saveCustomUserTest(){
+        Assertions.assertTrue(testObject.saveUser(testCustomUser));
+        testObject.deleteCustomUser("test@mail.com");
+    }
+
+    @Test
+    public void saveCustomUserTestFail(){
+        testObject.saveUser(testCustomUser);
+        Assertions.assertFalse(testObject.saveUser(testCustomUser));
```

Comment:
why does it fail?

Reply:
because user with the same email exist, and returns false like supposed.